src/Illuminate/Support/Collection.php

```diff
@@ -602,12 +602,12 @@ public function map(callable $callback)
     /**
      * Run an associative map over each of the items.
      *
-     * The callback should return an associative array with a single key/value pair.
+     * Map a collection persisting any changes made to keys.
```

Comment:
Whoops changed the wrong line, it was meant for the 'Run an associative map over each of the items.'